### DISCUSSION TITLE
Failed to setup Chromium

### DISCUSSION THREAD

**Discussion by Tekno68** [jef/streetmerchant#2349] [Help]
After typing npm install I got an error

Downloading Chromium r818858 - 132.1 Mb [====================] 100% 0.0s 
ERROR: Failed to set up Chromium r818858! Set "PUPPETEER_SKIP_DOWNLOAD" env variable to skip download.
[Error: Unknown system error -122: Unknown system error -122, write] {
  errno: -122,
  code: 'Unknown system error -122',
  syscall: 'write'
}

**Comment by jef** [comment]
Run `node ./node_modules/puppeteer/install.js`. This should solve your issue.

**Reply by Tekno68** [reply]
Chromium is already in /home/runner/streetmerchant-3/node_modules/puppeteer/.local-chromium/linux-856583; skipping download.

**Comment by WallisonLima** [comment]
Tente baixar o navegador diretamente com a instalação do puppeteer:
`npm install puppeteer`
Espero que ajude
